src/NuGet.Commands/RestoreCommand.cs

```diff
@@ -27,6 +29,9 @@ namespace NuGet.Commands
 {
     public class RestoreCommand
     {
+        // Temporary until we have RESX and loc
```

Comment:
Do we plan to loc this for RTM? the deadline for locking down strings is coming up.